lms/static/sass/multicourse/_courses.scss

```diff
@@ -257,160 +333,152 @@ $facet-background-color: #007db8;
-    @extend %ui-depth1;
-    position: relative;
-    margin: ($baseline*2) 0 ($baseline*3.5) 0;
-    box-shadow: 1px 2px 5px $black-t0;
-    border-top: 1px solid $black;
-    border-bottom: 2px solid $black;
-    background-color: $white;
-    max-height: ($baseline*100);
+  }
 
-    @include media($bp-tiny) {
-      @include span-columns(4);
-    }
+  @include media($bp-small) {
+    @include span-columns(4);
+  }
 
-    @include media($bp-small) {
-      @include span-columns(3);
-    }
+  @include media($bp-medium) {
+    @include span-columns(8);
+  }
 
-    @include media($bp-medium) {
-      @include span-columns(4);
-    }
+  @include media($bp-large) {
+    @include span-columns(4);
+  }
 
-    @include media($bp-large) {
-      @include span-columns(4);
-    }
+  @include media($bp-huge) {
+    @include span-columns(3);
+  }
 
-    @include media($bp-huge) {
-      @include span-columns(3);
-    }
+  &.phone-menu {
+    border: medium none;
+    padding: 0;
+    overflow: visible;
+  }
 
-    &.phone-menu {
-      border: medium none;
-      padding: 0;
-      overflow: visible;
-    }
+  &:before {
+    @include right(0);
+    position: absolute;
+    top: (-$baseline*0.15);
+    opacity: 0;
+    background-color: $white;
+    padding: ($baseline*2) ($baseline*0.75) 0 ($baseline*0.75);
+    width: ($baseline*2.5);
+    height: ($baseline/4);
+    content: "";
+  }
 
-    &:before {
-      @include right(0);
-      position: absolute;
-      top: (-$baseline*0.15);
-      opacity: 0;
-      background-color: $white;
-      padding: ($baseline*2) ($baseline*0.75) 0 ($baseline*0.75);
-      width: ($baseline*2.5);
-      height: ($baseline/4);
-      content: "";
-    }
+  .header-search-facets, .header-facet {
+    @extend %t-title6;
+    padding: ($baseline/2);
+    margin: 0;
+    color: $facet-text-color;
+    text-transform: none;
+  }
 
-    h2,
-    section {
-      @extend %t-icon5;
-      @extend %t-strong;
-      margin: 0 ($baseline/2);
-      border: medium none;
-      padding: ($baseline/2);
-      color: $facet-text-color;
-      font-family: $sans-serif;
-      text-transform: none;
-    }
+  .header-facet {
+    margin: 0 ($baseline/2);
+    padding: ($baseline/2) 0;
+  }
 
-    h3 {
-      @extend %t-icon6;
-      @extend %t-strong;
-      margin: 0 ($baseline/2) ($baseline/2) ($baseline/2);
-      color: $facet-text-color;
-      font-family: $sans-serif;
+  .search-facets-lists section {
+    border-top: 1px solid $courseware-button-border-color;
+  }
+
+  .facet-list {
+    @extend %ui-no-list;
+    @include clearfix();
+    padding-bottom: ($baseline/2); 
+  }
+
+  .facet-option {
+    @include float(left);
+    @include line-height(18.92);
+    @include transition(all $tmg-f2 ease-out 0s);
+    @include clearfix();
+    @extend %t-action3;
+    @extend %text-truncated;
+    border-radius: 0px;
+    opacity: 1;
+    width: 100%;
+    color: $facet-text-color;
+
+    &.collapse {
```

Comment:
collapse does not belong here I think, it is a class added to .facet-list (to hide or show more options).